plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/table/HudiTableFileSystemView.java

```diff
@@ -272,7 +272,8 @@ private void ensurePartitionLoadedCorrectly(String partition)
                         storePartitionView(partitionPathStr, new ArrayList<>());
                     }
                 }
-                catch (IOException e) {
+                catch (Exception e) {
+                    
```

Comment:
The style is wrong. Also, why do we need to change this exception? 
```
Error:  src/main/java/io/trino/plugin/hudi/table/HudiTableFileSystemView.java:[276] (regexp) RegexpMultiline: Line has trailing whitespace
```

Reply:
Sorry, I thought wrong.
Will update it and add test.